ES/ESmy009.xml

```diff
@@ -30,95 +30,1407 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   <idno>MY-009</idno>
```

Comment:
images should not be visible? if you need the viewer 
`<idno facs="MY/009/MY-009" n="115">MY-009</idno>`
(if images are uploaded)

Reply:
I do not know if they are uploaded (possibly not), but they should be visible.